src/test/java/core/basesyntax/service/impl/ReaderImplTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.Reader;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReaderImplTest {
+    private static final String PATH = "src/test/resources/fruits.csv";
+    private static List<String> expectedResult;
+    private static Reader reader;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        reader = new ReaderImpl();
+        expectedResult = List.of("type,fruit,quantity",
```

Comment:
expected value is used only in one test, so it can be placed in that particular test